src/lib/dom.iterable.d.ts

```diff
@@ -5,9 +5,17 @@ interface DOMTokenList {
 }
 
 interface NodeList {
+    keys(): IterableIterator<number>;
+    values(): IterableIterator<[number, Node]>;
+    entries(): IterableIterator<Node>;
+    forEach(callbackfn: (value: Node, index: number, listObj: NodeList) => void, thisArg?: any): void;

```

Comment:
I don't think that is correct. The return values for entries and values are swapped.  https://developer.mozilla.org/en-US/docs/Web/API/NodeList/entries